src/sdk/common/constants.ts

```diff
@@ -10,3 +10,32 @@ export enum HeaderNames {
 export const bufferPercent = 13; // Buffer in percent
 
 export const onRampApiKey = 'pk_prod_01H66WYDRFM95JBTJ4VMGY1FAX';
```

Comment:
Is this ok to stay in the file?

Reply:
This is the default key, which we have previously included in this file.